lib/querystring.js

```diff
@@ -179,7 +200,7 @@ function stringify(obj, sep, eq, options) {
     for (let i = 0; i < len; ++i) {
       const k = keys[i];
       const v = obj[k];
-      let ks = encode(stringifyPrimitive(k));
+      let ks = convert(k, encode);
```

Comment:
This could be simplified to: 

```suggestion
      let ks = encode(typeof k === 'string' ? k : '');
```

Reply:
It's not quite the same since `encodeStringified()` can bypass `encode()` for empty strings for example.